_tpl/_html-foot.tpl

```diff
@@ -18,54 +18,7 @@
 <script src="{{ url static_file='_js/fancybox/jquery.mousewheel-3.0.4.pack.js' }}" type="text/javascript"></script>
 <script src="{{ url static_file='_js/fancybox/jquery.fancybox-1.3.4.pack.js' }}" type="text/javascript"></script>
 <link rel="stylesheet" type="text/css" href="{{ url static_file='_js/fancybox/jquery.fancybox-1.3.4.css' }}" media="screen" />
-<a id="fancybox-trigger" href="#werbung-mitglied" style="display:none"></a>
-<script type="text/javascript">
-        function triggerWerbungPopup() {
-            $("#fancybox-trigger").fancybox({
```

Comment:
Well I was thinking not to remove it, just to remove js .. but If you think that this is must, I can do it?